docs/overview/proof-of-burn.md

```diff
@@ -0,0 +1,36 @@
+# Proof-of-Burn
+
+Proof-of-Burn (PoB) is the consensus algorithm used by the Koinos blockchain. It shares properties with both Proof-of-Work (PoW) and Proof-of-Stake (PoS). At a high level, Proof-of-Burn emulates the behavior of Proof-of-Work while keeping the concept of real world assets such as mining rigs virtualized on-chain.
+
+### Similarities to Proof-of-Work
+Like Proof-of-Work, a potential block producer will acquire hash power by sacrificing capital in order to have higher chances of mining blocks. The more hash power one has, the more likely they will produce a block.
+
+### Similarities to Proof-of-Stake
+Similar to Proof-of-Stake, mining blocks does not consume large amounts of electricity. It is therefore accessible to more participants to produce blocks competitively because profitability is not tied to hardware access or cheap electricity.
+
+### Key differences between Proof-of-Stake and Proof-of-Burn
+In traditional Proof-of-Stake blockchains a block producer stakes his coin to gain the privilege of producing blocks. The network ensures that the block producer is behaving as expected. If the network finds the block producer misbehaving, it will slash or deduct a penalty from their staked coin.
+
+Proof-of-Burn turns this paradigm upside down. Rather than staking coins and checking for misbehavior, Proof-of-Burn requires you to burn your coins and rewards you for good behavior. To participate you must effectively slash yourself and regain your initial capital plus interest by contributing the network.
+
+---
+## Advantages of virtual mining
+Because mining rigs are virtualized on chain, we remove the need to spend significant amounts of capital to purchase hardware from vendors. Because your chances to find blocks are not based on computational hash power you will not use exorbitant amounts of electricity to mine blocks.
```

Comment:
Previously you used, "on-chain". I believe the hyphenated form should only be used when "on-chain" is being used as an adjective. (e.g. an on-chain mining rig) But here, "chain", is the object of the preposition, "on".